packages/es-components/src/components/base/spinner/Spinner.md

```diff
@@ -0,0 +1,24 @@
+Spinners can be sized how you like, but their contents will always be an exact circle.
+
+They can be sized to the outer container:
+```
+<div style={{width: 100, height: 100}}>
+  <Spinner />
+</div>
+```
+
+You can manually set width and height:
+```
+<div>
+  <Spinner width="65px" height="65px" />
+</div>
+```
+
+
+You can set the size based on a percentage:
+```
+<div style={{width: 100, height: 100}}>
+  <Spinner width="50%" />
+</div>
+
```

Comment:
I think it will be helpful to call out that the component can be styled with css as any other component. We don't need to show any examples - we can even just link to my comment in the PR - but calling out that the component can be styled with whatever css tooling is used would help keep the user's code consistent.

Reply:
I added some clarification around how you can style it, but I wonder if it would be a little too much "how the sausage is made" to link back to this PR in the docs.